ui/src/components/form/QForm.js

```diff
@@ -49,7 +49,7 @@ export default Vue.extend({
 
       for (let i = 0; i < components.length; i++) {
         const comp = components[i]
-        const valid = comp.validate()
+        const valid = typeof comp.validate === 'function' ? comp.validate() : true
```

Comment:
comp is ensured to have validate() from this.getValidationComponents()

Reply:
My bad, it's checked.